src/templates/register.html

```diff
@@ -16,6 +16,6 @@ <h1>Register</h1>
         <input type="submit" value="Register" class="button">
     </form>
     {% if error %}
-        <p class="error-message">{{ error }}</p>
+        <p>{{ error }}</p>
```

Comment:
? The old one seemed better to me.

Reply:
Are you referring to removing the class? I don't have any CSS yet. It is a conscious choice as I'm trying to avoid my usual shenanigans of designing and implementing the UI first (and spending way too much time on selecting colours, gradients and doing pseudo 3D-effects). But again, this also displays my lack of planning ahead. The classes/ids should be included to make building the UI easier.